juneau-core/src/main/java/org/apache/juneau/internal/StringUtils.java

```diff
@@ -618,7 +618,9 @@ public static boolean isEmpty(String s) {
 	 * @return <jk>true</jk> if specified string is <jk>null</jk> or it's {@link #toString()} method returns an empty string.
 	 */
 	public static boolean isEmpty(Object s) {
-		return s == null || s.toString().isEmpty();
+		if( s == null ) return true;
+		if( s instanceof List ) return ((List) s).size() == 0;

```

Comment:
@clr-apache the only problem with that is `Collection` is what introduces `isEmpty()`, `Iterable` only provides a handle to the `Iterator`